lib/validator.js

```diff
@@ -192,13 +192,13 @@ Specberus.prototype.getSotDSection = function () {
     if (!$startH2 || !$endH2) return null;
     var started = false;
     $startH2.parent().children().each(function () {
-        if ($startH2[0] === this[0]) {
+        if ($startH2[0] === this) {
             started = true;
             return;
         }
         if (!started) return;
-        if ($endH2[0] === this[0]) return false;
-        $div.append(this.clone());
+        if ($endH2[0] === this) return false;
+        $div.append($(this).clone());
     });
```

Comment:
Argh... You want me to do a pass on everything else? We might want to rely on the second argument of the function passed as an argument (`.each(function (index, element) { /* ... */ })`) instead of `this`, I think it would be safer.